### PR TITLE
[Access] rollback boxo to 0.17

### PR DESCRIPTION
**Pull request by peterargue**
We're using a forked version anyway, so this doesn't change the actually run version.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -100,7 +100,7 @@ require (
 	github.com/golang/snappy v0.0.5-0.20220116011046-fa5810519dcb
 	github.com/gorilla/websocket v1.5.0
 	github.com/hashicorp/golang-lru/v2 v2.0.7
-	github.com/ipfs/boxo v0.19.0
+	github.com/ipfs/boxo v0.17.0
```
Comment (turbolent) [inline go.mod:103]: Best to use the same version as in the replace statement, `89bceff34bf1` (replace statement refers to https://github.com/onflow/boxo/pull/1, i.e. commit https://github.com/onflow/boxo/commit/f2477b92f4831221274c63960b5d0a65d84180bf; see parent)

**Review by turbolent — COMMENTED** [review]
Thanks!

**Comment by turbolent** [comment]
Please also roll back the changes in `insecure` and `integration`

**Review by turbolent — APPROVED** [review]
Thank you!

**Review by sideninja — APPROVED** [review]

**Pull request merged**